### PR TITLE
chore: Add screenshots to fastlane

### LINKED ISSUE #1790: Add screenshots to fastlane
**Describe the chore**
Add screenshots to fastlane images folder.

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1790

Changes:

Added screenshots to fastlane images folder.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Links in the docs can be updated. If the images in docs are more up to date, use them instead please

**Comment by ShridharGoel** [comment]
Images in docs are same as these.

**Comment by ShridharGoel** [comment]
> Links in the docs can be updated.

Which links?

**Comment by iamareebjamal** [comment]
> Images in docs are same as these.

Then why adding new images? Just move them and update the links in docs

**Comment by ShridharGoel** [comment]
Updated.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 Event management app for organizers using Open Event Platform
 
-<img height="256px" src="docs/images/organizer_app_logo.png" />
+<img height="256px" src="fastlane/metadata/android/en-US/images/icon.png" />
```
Comment (iamareebjamal) [inline README.md]: Revert

Comment (ShridharGoel) [reply README.md]: Done.

**Pull request merged**